paper/basic_training.tex

```diff
@@ -730,11 +731,170 @@ \subsubsection{Summary}
 \subsection{Barostats}
 \label{sec:barostats}
 \begin{itemize}
-\item What is it?
+\item Motivation
+\item Background
 \item Brief description of how it works
 \item Popular barostats
+\item Summary
 \end{itemize}
 
+\subsubsection{Motivation}
+When browsing the literature for many physical properties of various
+materials, you will observe that a multitude of the thermodynamic
+properties of interest are measured under some contstant temperature
+and pressure. Whether it is standard temperature and pressure (STP), or
+a more extreme value, many experimental studies are performed under
+these conditions. Termed the isothermal-isobaric ensemble, this
+might be one of the most popular ensembles for simulationists in
+general. As is the case with thermostats, if the pressure must be
+maintained in a simulation, a barostat algorithm will be needed to
+sample this ensemble. This section will review the background of the
+barostats, a general overview of how they work, and introduce some
+popular barostats.
+
+\subsubsection{Background and How They Work}
+Recall that in experimental studies, to control the pressure,
+some device must be used to control the volume of the system. By
+controlling the volume of the system, a constant pressure can be
+achieved\cite{tuckermanBook}. By allowing the system to
+exchange volume with the surroundings, the system is at
+a constant pressure\cite{ShellNotes}.
+
+
+For the purpose of molecular
+modeling, consider a system with a ficticious piston of some ficticious
```

Comment:
"fictitious", sorry for being spelling police.

Reply:
Also it will probably not be clear to the novice reader why we are talking about a fictitious piston as opposed to a real piston, especially when we refer to its mass. Maybe need to rephrase to clarify how the piston is involved.

Unrelatedly, in LaTeX the proper quotes are formatted as two open back quotes (top left of your keyboard) and then two single quotes to close (near your return key). Can't get markdown to display properly, but see elsewhere in the Tex.